frontend/scss/application.scss

```diff
@@ -0,0 +1,22 @@
+@import "bootstrap";
+@import "vendor";
+@import "components";
+
+.navbar-brand {
```

Comment:
these should go to bootstrap override I think, probably the bootstrap/_type.scss file.